pynucastro/rates/rate.py

```diff
@@ -1770,12 +1762,12 @@ def __init__(self, rate, compute_Q=False, use_pf=False):
             if len(self.rate.reactants) == len(self.rate.products):
                 prefactor += 0.0
             else:
-                F = (amu * k_B * 1.0e5 / (2.0*np.pi*hbar**2))**(1.5*(len(self.rate.reactants) - len(self.rate.products)))
+                F = (constants.m_u * constants.k / (2.0*np.pi*constants.hbar**2))**(1.5*(len(self.rate.reactants) - len(self.rate.products)))
```

Comment:
This fixes the failing python tests, and probably the C++ test too.
```suggestion
                F = (constants.m_u * constants.k * 1.0e9 / (2.0*np.pi*constants.hbar**2))**(1.5*(len(self.rate.reactants) - len(self.rate.products)))
```

Reply:
oh... this is clearer -- so the 1.e9 is from the fact that it is in units of 10**9 K?